BUILD_COMMANDLINE.md

```diff
@@ -8,7 +8,7 @@
    |----|----------------|
    |Java| Java Development Kit (JDK) 17 recommended (at least Java 8 at runtime to support Apache Ant™)|
    |Apache Ant|1.10.14|
-   |Verovio Toolkit|4.2.1|
+   |Verovio Toolkit|4.5.1|
```

Comment:
```suggestion
   |Verovio Toolkit|5.0.0|
```

Reply:
Thanks for the hint. The docker-mei is still on 4.5.1; we should update it to 5.0.0, too. OR even first and then here.

Reply:
Resolved in music-encoding/docker-mei#74